fcrepo-exts/fcrepo-doctor
# add support for fixing problems

# What does this Pull Request do?

Adds support for fixing problems. Currently, this just means fixing binary descriptions with invalid subjects.

# How should this be tested?

1. Run a version of Fedora prior to [this commit](https://github.com/fcrepo/fcrepo/commit/b05ed9400b0d4dc7943beb5f05a36500a820adb8)
2. Create binary descriptions with subjects like `<info:fedora/resource/fcr:metadata>`
3. Run `fcrepo-doctor` as described in the README
4. Confirm that the fixed RDF is correct and that the resources are usable in Fedora

# Interested parties

 @fcrepo/committers
